### PR TITLE
[CDAP-2592] update dependencies

### PR DESCRIPTION
**Pull request by elfenheart**
Distributed Singlenode: singlenode2012-1000
Secure: securecluster2452-1000

### CODE REVIEW

**Comment by elfenheart** [comment]
putting a hold on this.. There's still some issue with authentication

**Comment by elfenheart** [comment]
ready for review

File: cdap-ui/app/features/admin/controllers/namespace/create-ctrl.js [hunk 1]
```diff
@@ -33,6 +32,11 @@ angular.module(PKG.name + '.feature.admin')
             myNamespace.getList(true).then(function() {
               EventPipe.emit('namespace.update');
             });
+
+            $timeout(function () {
+              $modalInstance.close();
```
Comment (ajainarayanan) [inline cdap-ui/app/features/admin/controllers/namespace/create-ctrl.js]: Is ```$modalInstance.dismisAll()``` not working? is ```$timeout``` the only solution? Seems like a hack. Does the docs suggest anything?

Comment (elfenheart) [inline cdap-ui/app/features/admin/controllers/namespace/create-ctrl.js]: Tried that, and it's still throwing the $$nextSibling is null error

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**